lib/dpl/providers/elasticbeanstalk.rb

```diff
@@ -123,11 +124,11 @@ def update_app
       def wait_until_deployed
         msgs = []
         1.upto(20) { return if check_deployment(msgs) }
-        error 'Too many failures'
+        error 'Deploy status unknown due to timeout. Increase the wait_until_deployed_timeout option'
       end
 
       def check_deployment(msgs)
-        sleep 5
+        sleep wait_until_deployed_timeout / 20
```

Comment:
this looks great @cloudhary. my only nitpick would be that with the default 600 `wait_until_deployed_timeout / 20` would wait for 30 seconds. should we maybe check a little more often?

Reply:
That's a solid point, didn't think of it. I've just made the change. I'll go back and test it with the builds I have

Reply:
Both test builds above still behave as intended with this latest change. Thanks @svenfuchs for catching this!